src/jshint.js

```diff
@@ -4242,7 +4242,18 @@ var JSHINT = (function() {
       // ImportClause :: ImportedDefaultBinding
       this.name = identifier();
       addlabel(this.name, { type: "unused", token: state.tokens.curr });
-    } else if (state.tokens.next.id === "*") {
+      if (state.tokens.next.value === ",") {
+        // ImportClause :: ImportedDefaultBinding , NameSpaceImport
```

Comment:
Sure, `ImportedDefaultBinding` allows writing a shorthand for the special `"default"` export name.

So say I have a module `"foo"`:

```js
export var x = "x";
export default "default";
```

In another file, I could *import ImportedDefaultBinding , NameSpaceImport*

```js
import fooDefault, * as fooAll from "foo"
console.log(fooDefault); // "default"
console.log(fooAll); // { default: "default", x: "x" }
```

Or I could *import ImportedDefaultBinding , NamedImports*

```js
import fooDefault, { x as fooX } from "foo"
console.log(fooDefault); // "default"
console.log(fooX); // "x"
```